client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.js

```diff
@@ -0,0 +1,216 @@
+            el.row.original.username === removingUserOperation.userName;
+          const deleteOperationFailed =
+            removingUserOperation.error &&
+            el.row.original.username === removingUserOperation.userName;
+          const removable =
+            !deleteOperationOccuring && el.row.original.role !== 'PI';
+
+          return (
+            <>
+              {deleteOperationFailed && (
+                <Message type="error">
+                  We were unable to delete the user.
+                </Message>
+              )}
+              {deleteOperationOccuring && <LoadingSpinner placement="inline" />}
+              {removable && (
+                <Button
+                  color="link"
+                  disabled={removingUserOperation.loading}
+                  onClick={e => {
+                    dispatch({
+                      type: 'REMOVE_USER_FROM_TAS_PROJECT',
+                      payload: {
+                        projectId,
+                        id: el.row.original.username
+                      }
+                    });
+                  }}
+                >
+                  Remove
+                </Button>
+              )}
+            </>
+          );
+        }
+      }
+    ],
+    [rawData, removingUserOperation]
+  );
+  const {
+    getTableProps,
+    getTableBodyProps,
+    rows,
+    prepareRow,
+    headerGroups
+  } = useTable({
+    columns,
+    data
+  });
+  return (
+    <table styleName="manage-team-table" {...getTableProps()}>
+      <thead>
+        {headerGroups.map(headerGroup => (
+          <tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <th {...column.getHeaderProps()}>{column.render('Header')}</th>
+            ))}
+          </tr>
+        ))}
+      </thead>
+      <tbody {...getTableBodyProps()}>
+        {rows.map(row => {
+          prepareRow(row);
+          return (
+            <tr>
+              {row.cells.map(cell => (
+                <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+              ))}
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+  );
+};
+AllocationsManageTeamTable.propTypes = {
+  rawData: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  projectId: PropTypes.number.isRequired
+};
+
+const AllocationsManageTeamModal = ({
+  isOpen,
+  toggle,
+  projectId,
+  projectName
+}) => {
+  const dispatch = useDispatch();
+  const { teams, loadingUsernames, search } = useSelector(
+    state => state.allocations
+  );
+  const addUserOperation = useSelector(
+    state => state.allocations.addUserOperation
+  );
+  const error = addUserOperation.error || search.error;
+  useEffect(() => {
+    dispatch({
+      type: 'ALLOCATION_OPERATION_REMOVE_USER_INIT'
+    });
+  }, [isOpen]);
+
+  const isLoading =
+    loadingUsernames[projectId] && loadingUsernames[projectId].loading;
+
+  const onAdd = useCallback(
```

Comment:
there is a bug where you can select someone.  then unselect them where the field is blank BUT then
* add button is enabled
* clicking add button will add the user you previously selected to project

Reply:
Error is still present, looking into it now.